client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Set up DCR and BTC wallets and register with the DEX.
+# dcrdex, dexc, and the wallet simnet harnesses should all be running before
+# calling this script.
+set -e
+echo initializing
+./dexcctl -p abc init
+echo configuring Decred wallet
+./dexcctl -p abc -p abc newwallet 42 ~/dextest/dcr/alpha/w-alpha.conf '{"account":"default"}'
+echo configuring Bitcoin wallet
+./dexcctl -p abc -p abc newwallet 0 ~/dextest/btc/harness-ctl/alpha.conf '{"walletname":"gamma"}'
+echo registering with DEX
+./dexcctl -p abc register 127.0.0.1:7232 100000000 ~/.dcrdex/rpc.cert
```

Comment:
The simnet test dcrdex is on port 17273 and certificate is at ~/dextest/dcrdex/rpc.cert

Reply:
Good point about the cert, but dcrdex doesn't change the port, at least not automatically, depending on net.  Same with dexc BTW, and I'd welcome resolutions to that.

Reply:
> The simnet test dcrdex is on port 17273 and certificate is at ~/dextest/dcrdex/rpc.cert

Good point. I've only been using the dcrdex simnet server for the core `-tags harness` test, but I'll start using it for manual testing too.

Reply:
Yeah, I haven't been using any of the harnesses when not specifically testing them because I hate having a folder called "dextest" in my home, but I guess I'll just have to get over it because it's not worth keeping up with changes.

Reply:
So a problem with using that harness for general testing is the 1min/30sec lock times.  I don't think we should encourage using the harness for general testing, especially interactive, unless we make the dex/testing/dcrdex/harness.sh lock times longer by default

Reply:
That is, we should revert this change to client/cmd/dexcctl/simnet-setup.sh or we should modify dex/testing/dcrdex/harness.sh with a flag to change the lock times only if requested.